confd/vars/main.yaml

```diff
@@ -24,6 +24,8 @@ kilda_victoriametrics_host: "victoriametrics.pendev"
 kilda_victoriametrics_write_port: "4242"
 kilda_victoriametrics_read_port: "8428"
 kilda_victoriametrics_path: ""
+kilda_victoriametrics_read_path: ""
+victoria_read_url_prefix: "http"
```

Comment:
I'd say that this is rather 'scheme' than 'url prefix'.

Reply:
Rename it, right after will test it on Rigel.

Reply:
renamed